pkg/bot.go

```diff
@@ -497,23 +500,38 @@ func (b *bot) deletePreviewSite(args ResponseRequest, verboseErrors bool) string
 	return strings.Join(msgs, "\n\n")
 }
 
-func (b *bot) hasPreviewSiteCapability(email, org string) (bool, error) {
+func (k kubeClients) getWorkspace(org string) string {
+	ns, err := k.nsLister.Get(org)
+	if err != nil {
+		klog.Errorf("failed to find namespace %v: %v", org, err)
+		return org
+	}
+	// for legacy organizations, ws == namespace == org
+	if ns.Labels["ddev.live/displayname"] != "" && ns.Labels["ddev.live/subscriptionstub"] != "" {
```

Comment:
I need to change one of these constants prior to consumption however:
https://github.com/drud/api-common/blob/main/utils/workspace.go#L10

for later...